ui/src/router/components/form/validation/schema.js

```diff
@@ -92,7 +92,9 @@ const routerNameRegex = /^[a-z0-9-]*$/,
   kafkaBrokersRegex =
     /^([a-z]+:\/\/)?\[?([0-9a-zA-Z\-%._:]*)\]?:([0-9]+)(,([a-z]+:\/\/)?\[?([0-9a-zA-Z\-%._:]*)\]?:([0-9]+))*$/i,
   kafkaTopicRegex = /^[A-Za-z0-9_.-]{1,249}/i,
-  trafficRuleNameRegex = /^[A-Za-z\d][\w\d \-()#$%&:.]*[\w\d\-()#$%&:.]$/;
+  trafficRuleNameRegex = /^[A-Za-z\d][\w\d \-()#$%&:.]*[\w\d\-()#$%&:.]$/,
```

Comment:
The form validation seems to have a regression - it's no longer scrolling back to the top of the section which has the errors:

https://user-images.githubusercontent.com/23465343/198171085-729198bf-b47a-4640-9305-07bdc674a256.mov

Reply:
I see the same behaviour in dev, not sure what triggers the scrolling back, will check it.

Reply:
If i understood it correctly, for [StepWizardHorizontal](https://github.com/gojek/mlp/blob/main/ui/packages/lib/src/components/multi_steps_form/StepsWizardHorizontal.js), there is no scrolling unlike the [AccordionForm](https://github.com/gojek/mlp/blob/main/ui/packages/lib/src/components/accordion_form/AccordionForm.js)  which has the [AccordionFormScrollController](https://github.com/gojek/mlp/blob/main/ui/packages/lib/src/components/accordion_form/AccordionFormScrollController.js) which does the scrolling to section `onSubmit`. 

Just that it usually isn't that common for a Step to be that long to notice it all along i guess.

Reply:
I didn't realise that. TIL. Future improvement to the `StepWizardHorizontal`, I suppose. 😅  Thanks for digging into this!